.github/workflows/ci.yaml

```diff
@@ -0,0 +1,50 @@
+name: ci
+
+on:
+  pull_request:
+    branches:
+      - '**'
+  push:
+    branches:
+      - master
+      - main
+      - 'release-*'
+
+jobs:
+  mri:
```

Comment:
I guess this is not an intentional naming.

Reply:
Thanks. Fixed https://github.com/woothee/woothee-php/pull/51/commits/9c9fd3fd84fb4cd55f1bcfe5ed86864681926fd9